assets/js/googlesitekit/datastore/user/feature-tours.js

```diff
@@ -141,6 +234,32 @@ const baseSelectors = {
 		return state.dismissedTours;
 	},
 
+	/**
+	 * Gets a list of tour objects that qualify for the given view context.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state       Data store's state.
+	 * @param {string} viewContext View context.
+	 * @return {(Object[]|undefined)} Array of qualifying tour objects
+	 *                                `undefined` while readiness is being resolved.
+	 */
+	getReadyTours( state, viewContext ) {
+		return state.readyTours[ viewContext ];
+	},
+
+	/**
+	 * Gets a list of all tour objects.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {Object[]} Array of tour objects.
+	 */
+	getTours( state ) {
+		return state.tours;
+	},
```

Comment:
Let's not add this selector, as it shouldn't be necessary. We only need to be able to get the relevant tours for a context, not all tours that are generally available. If we need it at some point, we can still add it.

Reply:
It's needed by the resolver for `get(Ready|Current)FeatureTours`. I'll mark it as `@private` for now, but unfortunately the only alternative is reaching into `state` directly which we don't do anywhere else.